theme.json

```diff
@@ -0,0 +1,413 @@
+    "border": {
+      "color": false,
+      "radius": false,
+      "style": false,
+      "width": false
+    },
+    "color": {
+      "text": false,
+      "background": false,
+      "link": false,
+      "custom": false,
+      "defaultPalette": false,
+      "defaultGradients": false,
+      "defaultDuotone": false,
+      "customGradient": false,
+      "customDuotone": false,
+      "palette": [
+        {
+          "slug": "text",
+          "color": "var(--wp--custom--text--secondary)",
+          "name": "text"
+        },
+        {
+          "slug": "heading",
+          "color": "var(--wp--custom--text--primary)",
+          "name": "heading"
+        },
+        {
+          "slug": "background",
+          "color": "var(--wp--custom--background--primary)",
+          "name": "background"
+        }
+      ],
+      "gradients": []
+    },
+    "custom": {
+      "body": {
+        "1": {
+          "font-family": "Aktiv Grotesk VF",
+          "font-weight": 400,
+          "line-height": "140%",
+          "font-size": "16px"
+        },
+        "2": {
+          "font-family": "Aktiv Grotesk VF",
+          "font-weight": 400,
+          "line-height": "140%",
+          "font-size": "18px"
+        }
+      },
+      "heading": {
+        "1": {
+          "font-family": "Aktiv Grotesk VF",
+          "font-weight": 500,
+          "line-height": "clamp(1.1em, calc(1.25em + -0.4vw), 1.2em)",
+          "font-size": "clamp(1.07rem, calc(1.02rem + 0.25vw), 1.28rem)",
+          "letter-spacing": "-0.01em"
+        },
+        "2": {
+          "font-family": "Aktiv Grotesk VF",
+          "font-weight": 500,
+          "line-height": "clamp(1.1em, calc(1.25em + -0.4vw), 1.2em)",
+          "font-size": "clamp(1.14rem, calc(1.04rem + 0.47vw), 1.53rem)",
+          "letter-spacing": "-0.01em"
+        },
+        "3": {
+          "font-family": "Aktiv Grotesk VF",
+          "font-weight": 500,
+          "line-height": "clamp(1.1em, calc(1.25em + -0.4vw), 1.2em)",
+          "font-size": "clamp(1.30rem, calc(1.08rem + 1.09vw), 2.20rem)",
+          "letter-spacing": "-0.01em"
+        },
+        "4": {
+          "font-family": "Aktiv Grotesk VF",
+          "font-weight": 500,
+          "line-height": "clamp(1.1em, calc(1.25em + -0.4vw), 1.2em)",
+          "font-size": "clamp(1.38rem, calc(1.08rem + 1.52vw), 2.64rem)",
+          "letter-spacing": "-0.01em"
+        },
+        "5": {
+          "font-family": "Aktiv Grotesk VF",
+          "font-weight": 500,
+          "line-height": "clamp(1.1em, calc(1.25em + -0.4vw), 1.2em)",
+          "font-size": "clamp(1.57rem, calc(1.04rem + 2.69vw), 3.81rem)",
+          "letter-spacing": "-0.01em"
+        }
+      },
+      "space": {
+        "padding": {
+          "global": "clamp(1.00rem, calc(0.40rem + 3.01vw), 3.50rem)"
+        },
+        "block": {
+          "gap": "clamp(1.50rem, calc(1.38rem + 0.60vw), 2.00rem)"
+        }
+      },
+      "alignment": {
```

Comment:
If the application has a max width we'll need to add a full alignment token. Otherwise full width will always span the entire viewport.

Reply:
I purposefully left that out as it wasn't being used outside the settings or styles in the `theme.json`. I figured if that is needed, then it could always be added during the import process (bridge tool) and can then be used as necessary within the `theme.json`.